src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageDetails.tsx

```diff
@@ -17,60 +17,60 @@ const WorkPackageDetails: React.FC<WorkPackageDetailsProps> = ({ workPackage })
   return (
     <PageBlock title={'Work Package Details'} headerRight={<WbsStatus status={workPackage.status} />}>
       <Grid container spacing={1}>
-        <Grid item xs={2} md={2}>
```

Comment:
ok i think the layout should be:

project lead | start date | progress
project man | end date  | expected progress
duration      | timeline status